package.json

```diff
@@ -16,7 +16,8 @@
     "coverage": "nyc npm test && npm run generate-lcov && npm run coveralls-coverage && npm run codeclimate-coverage",
     "codeclimate-coverage": "codeclimate-test-reporter < lcov.info",
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
+    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit",
+    "testdev": " mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
```

Comment:
@davidfrank96 We don't need this script for `testdev`. Every test should be run in the test environment. We normally won't want to update the development database with test records.